content/blog/posts/o3de-graphics-atom-intro.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Exploring the world of graphics via Atom, O3DE’s renderer
+date: 2022-06-23
+slug: graphics-atom-intro
+author: Judy Ng
+blog_img: ""
+full_img: ""
+---
+Hello all! I’m Judy, and I’m an intern for the Open 3D Engine (O3DE) Atom team. I interned with Atom in the summer of 2021 and came back to the team to intern again this summer 2022! So far this summer, I’ve learned graphics fundamentals and applied that knowledge to write a custom shader and make my own material type. Here’s some details about my journey learning about graphics and Atom.
+
+## Learning through doing
+
+Having minimal experience with graphics before, I took the phenomenon of the “computer display” for granted. However, from working with shaders, I gained an appreciation for the complexity of displaying these millions of pixels that create images on a screen. Diving deeper into Atom, with complicated 3D graphics, I learned about *passes*, *materials*, and *shaders*. While I’d heard of these terms before, I didn’t know what exactly they meant, but the best way I learned them was just from working with them.
```

Comment:
Suggested rewrite: ... but the best way I learned about them was simply through my hands-on experience in working with them.